src/ray/rpc/worker/direct_actor_client.h

```diff
@@ -75,9 +78,9 @@ class DirectActorClient : public std::enable_shared_from_this<DirectActorClient>
               if (seq_no > max_finished_seq_no_) {
                 max_finished_seq_no_ = seq_no;
               }
-              rpc_bytes_in_flight_ -= task_size;
-              RAY_CHECK(rpc_bytes_in_flight_ >= 0);
             }
+            rpc_bytes_in_flight_ -= task_size;
+            RAY_CHECK(rpc_bytes_in_flight_ >= 0);
```

Comment:
Isn't it important to keep this under the lock guard?

Reply:
Oops, this was leftover from a previous PR where I'd already removed it. I'll remove it.